tests/core/CoreTest.py

```diff
@@ -667,9 +681,123 @@ def test_custom_executor_closed_after_run(self):
         for bears in [set(), {bear}]:
             executor = ThreadPoolExecutor(max_workers=1)
 
-            self.execute_run(bears, executor)
+            self.execute_run(bears, executor=executor)
 
             # Submitting new tasks should raise an exception now.
             with self.assertRaisesRegex(
                     RuntimeError, 'cannot schedule new futures after shutdown'):
                 executor.submit(lambda: None)
+
+
+class CoreCacheTest(CoreTestBase):
+    def setUp(self):
```

Comment:
There should be one blank line before the function declaration. :)

Reply:
You sure that it's necessary for PEP8? I read different opinions regarding this^^ I personally favour more this style^^

Reply:
Well it is not explicitly mentioned like this in the documentation. But it is the preferred style IMO. :)